provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/publish.yml

```
@@ -114,10 +118,17 @@ jobs:
       with:
         tools: pulumictl, pulumicli, #{{ range $index, $element := .Config.Languages }}##{{if $index}}#, #{{end}}##{{ $element }}##{{end}}#
     - name: Publish SDKs
+      if: inputs.skipJavaSdk == false
       uses: pulumi/pulumi-package-publisher@1c0359ba74243cf6651efacfd839c751d8ff87e2 # v0.0.20
       with:
         sdk: #{{ .Config.Publish.SDK }}#
         version: ${{ inputs.version }}
+    - name: Publish SDKs (except Java)
+      if: inputs.skipJavaSdk == true
+      uses: pulumi/pulumi-package-publisher@1c0359ba74243cf6651efacfd839c751d8ff87e2 # v0.0.20
+      with:
+        sdk: #{{ .Config.Publish.SDK }}#,!java
```

Nit: As suggested by https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#example, it looks like we could make this more concise:

```suggestion
        sdk: #{{ .Config.Publish.SDK }}#${{ inputs.skipJavaSdk && ',!java' || '' }}
```

Otherwise, LGTM.